Electron/Electron12/protected/package.json

```diff
@@ -68,6 +66,8 @@
     "@electron-forge/maker-zip": "~6.0.0-beta.54",
     "@electron-forge/plugin-webpack": "6.0.0-beta.54",
     "@marshallofsound/webpack-asset-relocator-loader": "~0.5.0",
+    "@preemptive/jsdefender-core": "file:assets/preemptive-jsdefender-core-2.6.0.tgz",
+    "@preemptive/jsdefender-webpack-plugin": "file:assets/preemptive-jsdefender-webpack-plugin-2.6.0.tgz",
```

Comment:
please revert this change, it's left for edit intentionally

Reply:
👍